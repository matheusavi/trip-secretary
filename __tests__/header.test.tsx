import { render } from "@testing-library/react";
import Header from "../app/header";

it("renders header unchanged", () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
});
