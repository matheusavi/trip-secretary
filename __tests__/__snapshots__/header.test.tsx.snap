// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header unchanged 1`] = `
<div>
  <header
    class="flex flex-row w-full justify-center py-2 bg-primary border-b-2 text-primary-foreground"
  >
    <div
      class="flex justify-between flex-row gap-2 max-w-screen-xl flex-1 items-center mx-2 2xl:mx-0"
    >
      <h2
        class="text-3xl font-bold"
      >
        Trip Secretary
      </h2>
      <div
        class="flex flex-row items-center cursor-pointer"
      >
        <svg
          aria-hidden="true"
          aria-label="User profile"
          class="h-10 flex-shrink-0 dark:drop-shadow-[0_0_0.3rem_#ffffff70]"
          data-slot="icon"
          fill="currentColor"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M18.685 19.097A9.723 9.723 0 0 0 21.75 12c0-5.385-4.365-9.75-9.75-9.75S2.25 6.615 2.25 12a9.723 9.723 0 0 0 3.065 7.097A9.716 9.716 0 0 0 12 21.75a9.716 9.716 0 0 0 6.685-2.653Zm-12.54-1.285A7.486 7.486 0 0 1 12 15a7.486 7.486 0 0 1 5.855 2.812A8.224 8.224 0 0 1 12 20.25a8.224 8.224 0 0 1-5.855-2.438ZM15.75 9a3.75 3.75 0 1 1-7.5 0 3.75 3.75 0 0 1 7.5 0Z"
            fill-rule="evenodd"
          />
        </svg>
        <svg
          aria-hidden="true"
          aria-label="User options"
          class="flex-shrink-0 h-5"
          data-slot="icon"
          fill="currentColor"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M12.53 16.28a.75.75 0 0 1-1.06 0l-7.5-7.5a.75.75 0 0 1 1.06-1.06L12 14.69l6.97-6.97a.75.75 0 1 1 1.06 1.06l-7.5 7.5Z"
            fill-rule="evenodd"
          />
        </svg>
      </div>
    </div>
  </header>
</div>
`;
